recipes/ipopt/LICENSE

```diff
@@ -0,0 +1,87 @@
+Eclipse Public License - v 1.0
```

Comment:
This doesn't need to be in the recipe as `license_file` is able to get this from the source.